ui/src/app/components/runs/filters/status-filter/status-filter.component.ts

```diff
@@ -0,0 +1,46 @@
+import {AfterViewInit, Component, Input, OnDestroy} from '@angular/core';
+import {Subject} from "rxjs";
+import {ClrDatagridFilterInterface} from "@clr/angular";
+import {DagRunModel} from "../../../../models/dagRuns/dagRun.model";
+
+@Component({
+  selector: 'app-status-filter',
+  templateUrl: './status-filter.component.html',
+  styleUrls: ['./status-filter.component.scss']
+}) export class StatusFilterComponent implements ClrDatagridFilterInterface<DagRunModel>, AfterViewInit, OnDestroy {
+  @Input() removeFiltersSubject: Subject<any>;
+  @Input() property: string;
+  @Input() statuses: {name: string, color: string, shape: string}[];
```

Comment:
This tuple is worth an own interface imo, otherwise you'll have to edit multiple places if you change it

Reply:
fixed